contents/Documentation/Podfile

```diff
@@ -0,0 +1,14 @@
+target 'Documentation' do
+  platform :osx, '10.14'
+  use_frameworks!
+
+  pod "Bow", "~> 0.3.0"
```

Comment:
maybe for documentation, we should delete `, "~> 0.3.0"`to keep it updates to the last version - we should review how to compile using a determined branch/version as we did in playground generation.

Reply:
Exactly. Right now we can resort to the latest published version, but we will need to adjust accordingly to compile against the last commit on the branch that is been pushed.